fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/impl/KeyValueStoreDimension.java

```diff
@@ -475,7 +475,7 @@ public void addAllDimensionRows(Set<DimensionRow> dimensionRows) {
                 indexRows.put(rowIdKey, new Pair<>(dimensionRow, dimensionRowOld));
 
             } catch (IOException e) {
-                LOG.error("Exception while adding dimension row {}", e);
+                LOG.error("Exception while adding dimension row", e);
```

Comment:
https://sonarcloud.io/project/issues?id=yahoo_fili&issues=AXPLWYhWIUmnqh_n8dv8&open=AXPLWYhWIUmnqh_n8dv8
> Not enough arguments

When the final argument is a Throwable it generates a stack trace and it is not used for formatting.  This maintains current functionality, but was original intent to show just the exception?
```java
LOG.error("Exception while adding dimension row {}", e.toString());
```

Reply:
The original intent was to make sure we didn't lose the precise logging associated with the error.
Stack trace is a very nice to have.